react-hot-button/my-app/src/App.tsx

```diff
@@ -0,0 +1,14 @@
+import './App.css';
+import HotButton from './hotbutton.tsx';
+import './index.css';
+
+export function App() {
+  return (
+    <>
+      <h1>Hot Button</h1>
+      <HotButton></HotButton>
```

Comment:
If there are no children, you should use the self-closing style:
`<HotButton />`

It's not a big deal, but experienced developers will think your code looks amateur.